pydy/viz/scene.py

```diff
@@ -460,9 +595,37 @@ def remove_static_html(self, force=False):
         else:
             print("aborted!")
 
+    def copy_resources(self, silent=False):
+        """
+        Copies simulation and static files to a directory pointed
+        by self.static_url attribute.
+        It overwrites if such directory is already present.
+        """
+        dst = self.static_url
+        if os.path.exists(dst):
+            if not silent:
+                print("Cleaning previous static data")
+            distutils.dir_util.remove_tree(dst)
+
+        src = os.path.join(os.path.dirname(__file__), 'static')
+
+        if not silent:
+            print("Copying static data.")
+        distutils.dir_util.copy_tree(src, dst)
+
+        if not silent:
+            print("Copying Simulation data.")
+        try:
+            shutil.move(os.path.join(os.getcwd(), self._scene_json_file),
+                        os.path.join(self.static_url, self._scene_json_file))
+            shutil.move(os.path.join(os.getcwd(), self._simulation_json_file),
+                        os.path.join(self.static_url, self._simulation_json_file))
+        except IOError:
+            pass
+
     def display(self):
         """Displays the scene in the default webbrowser."""
-        self.create_static_html()
+        self.copy_resources()
```

Comment:
the `create_static_html` method should also use this design. It should copy over the files in the cwd directory `static` and then call `_generate_json` and place those files in the directory. Then the `static` dir contains the whole "app" for that one simulation that the user may want to hold on to.